bin/docs.py

```diff
@@ -42,12 +42,14 @@
 
     docs.write("_This file was generated by the `bin/docs.py` script on %s._\n\n" % ymd)
 
+    docs.write("Crediting Who's On First is recommended and linking back to the License is required. For example:\n\n> Data from Who's On First. [License](https://whosonfirst.org/docs/sources/list/).\n\nThe Who's On First dataset is both original work and a modification of existing open data (sources listed below). Some of those open data projects do require attribution. We have listed all sources below in the full [license file](https://whosonfirst.org/docs/sources/list/).\n\n")
```

Comment:
Please move this up above `This file was generated...`

Reply:
taken care of in f44b85f and 9d41f48